datastore/src/main/java/org/jboss/pnc/datastore/limits/rsql/EmptySortInfo.java

```diff
@@ -35,6 +35,10 @@ public List<OrderInfo<T>> orders() {
     }
 
     private static <T> Expression<?> idOrder(Root<T> root) {
-        return root.get("id");
+        try {
+            return root.get("id");
+        } catch (IllegalArgumentException ex) {
+            return root;
+        }
```

Comment:
Had to add this workaround, since in the `DeliverableArtifact`, there's no column called "id" and would crash it in runtime.